backend/tests/Designer.Tests/Controllers/ApiTests/DesignerEndpointsTestsBase.cs

```diff
@@ -118,6 +118,25 @@ protected async Task CopyOrgRepositoryForTest(string developer, string org, stri
             TestOrgPath = await TestDataHelper.CopyOrgForTest(developer, org, repo, targetOrg, targetRepository);
         }
 
+        /// <summary>
+        /// Copies a repository from the test repositories to a temporary location for testing.
+        /// Limitation is that test org has to be created through <see cref="CopyOrgRepositoryForTest"/>
+        /// </summary>
+        /// <param name="developer">Username of developer.</param>
+        /// <param name="org">Organisation short name.</param>
+        /// <param name="repo">Repository name.</param>
+        /// <param name="targetOrg">Test organisation name.</param>
+        /// <param name="targetRepository">test repository name.</param>
+        /// <exception cref="InvalidOperationException"></exception>
+        protected async Task AddRepositoryToTestOrg(string developer, string org, string repo, string targetOrg, string targetRepository)
+        {
+            if (TestOrgPath is null)
+            {
+                throw new InvalidOperationException("Organization is not created for test.");
```

Comment:
Vi tar sikte på å bruke britisk engelsk i Altinn😊 
```suggestion
                throw new InvalidOperationException("Organisation is not created for test.");
```